ceph-ansible.spec.in

```diff
@@ -35,6 +35,14 @@ for f in ansible.cfg *.yml *.sample group_vars roles library plugins infrastruct
   cp -a $f %{buildroot}%{_datarootdir}/ceph-ansible
 done
 
+# Strip coreos files.
+# These are unneeded on RPM-based distros, and the playbooks download random
+# things from around the internet.
+pushd %{buildroot}%{_datarootdir}/ceph-ansible
+  rm -r roles/ceph-common-coreos
```

Comment:
What would be the purpose of `-f` there?

If the `rm` operation fails, it points to a deeper issue: either a) the build system itself is broken, or b) maybe the files do not exist in the tree any more, and we would update this code so it no longer deletes non-existent files.